### PR TITLE
Add ctrl-tab and ctrl-shift-tab.

### PR DESCRIPTION
**Pull request by bwinton**
I really missed these two…

### CODE REVIEW

File: src/browser-frontend/sagas/keyboard-shortcuts-action-handlers.js [hunk 1]
```diff
@@ -38,6 +38,34 @@ function* handleAccelT() {
   yield put(PagesModelActions.addPage());
 }
 
+function* handleCtrlTab() {
+  const selectedPageId = yield select(UIPagesSelectors.getSelectedPageId);
+  const pages = yield select(UIPagesSelectors.getPageIdsInDisplayOrder);
+  const pageIndex = pages.findIndex(id => id === selectedPageId);
```
Comment (victorporof) [inline src/browser-frontend/sagas/keyboard-shortcuts-action-handlers.js]: Nit: Consider writing a dedicated selector for this: getSelectedPageIndex.

File: src/browser-frontend/sagas/keyboard-shortcuts-action-handlers.js [hunk 2]
```diff
@@ -38,6 +38,34 @@ function* handleAccelT() {
...
+  let newIndex = pageIndex + 1;
+  if (newIndex >= pages.count()) {
+    newIndex = 0;
+  }
+  if (pageIndex === newIndex) {
+    return;
+  }
+  yield put(PagesModelActions.setSelectedPage({ pageId: pages.get(newIndex) }));
```
Comment (victorporof) [inline src/browser-frontend/sagas/keyboard-shortcuts-action-handlers.js]: Nit: Consider a selector for this as well: getPageIdAtIndex

**Review by victorporof — APPROVED** [review]
love this!

**Comment by bwinton** [comment]
So, there was a bunch of code that kinda relied on the various internal bits of data, and I ended up moving it all into a couple of selectors.  I'm finding it to be a tricky balance trying to figure out what should be in a selector and what shouldn't.

**Comment by bwinton** [comment]
(Hah!  Retriggered, and the build worked. 😄)

**Comment by victorporof** [comment]
It's always a difficult task to see how much code should be in selectors vs. other places. I use ease of testing as my guideline. Your approach makes testing even easier, so r+ shipit.

**Review by victorporof — APPROVED** [review]

**Pull request merged**